mm2src/mm2_main/tests/docker_tests/mod.rs

```diff
@@ -12,4 +12,4 @@ mod swaps_file_lock_tests;
 
 // dummy test helping IDE to recognize this as test module
 #[test]
-fn dummy() { assert!(true) }
+fn dummy() {}
```

Comment:
I don't use IDEs, can anyone test this?

cc @caglaryucekaya @shamardy @borngraced @laruh